tests/core/test_dialogue.py

```diff
@@ -184,8 +190,9 @@ def test_to_dict_d1(dialogue_history_1: Dialogue) -> None:
     assert len(dialogue_dict_1.get("conversation")) == 5
     utterance_1 = dialogue_dict_1.get("conversation")[0]
     assert utterance_1["utterance"] == "Hello"
+    assert utterance_1.get("dialogue_acts") is None
+    assert utterance_1.get("annotations") is None
```

Comment:
Please also test the case where we also have non-dialogue act annotations.

Reply:
Done in `test_to_dict_d2`.